pkg/image/pull.go

```diff
@@ -0,0 +1,87 @@
+package image
```

Comment:
I’m not sure `image` is the right name for this subpackage — it might be. But naming the file `pull.go` just isn’t a good match.